infra/launch.py

```diff
@@ -102,8 +125,33 @@ def start_tpu_vm(tpu_name, *, tpu_type, capacity_type, version, zone, autodelete
         pass
     else:
         raise ValueError(f"Unknown capacity type: {capacity_type}")
+
+    if node_count == 1:
+        command.append(f"--node-id={tpu_name}")
+    else:
+        command.append(f"--node-count={node_count}")
```

Comment:
set `node-prefix`  to tpu_name?

Reply:
The tpu name will automatically become the node prefix.

Reply:
but you're not appending the tpu name?

Reply:
oh i see because it's the queued resource name.